src/plugins/home/public/application/components/home_app.js

```diff
@@ -32,6 +32,7 @@ import React from 'react';
 import { I18nProvider } from '@osd/i18n/react';
 import PropTypes from 'prop-types';
 import { Home } from './home';
+import { Homepage } from './homepage';
```

Comment:
nitpick: 

for consistency, i think we should do what we did with Discover page. we converted the legacy to `discover_legacy` and the new content was `discover`. 

it will be easier to follow code wise so that `/legacy` maps to `home/public/application/components/home_legacy.js` or `home/public/application/components/legacy/home.js`. 

vs knowing `/` maps to `home/public/application/components/homepage` and `/legacy` mapping to `home/public/application/components/home.js`

Reply:
I moved a bunch of stuff into a `legacy` directory, but now I'm wondering if I should move all of the stuff out of the `homepage` directory or if it would be nice to keep it like that for organization?

Reply:
I find it easier to throw things into a folder when they are more than 2. This will help make sure nothing is accidentally consumed by someone else. `legacy_` vs `legacy/` is really just a visual thing.